VK_App/VK_App/View/OtherGroupsViewCell.swift

```diff
@@ -7,8 +7,8 @@ import UIKit
 final class OtherGroupsViewCell: UITableViewCell {
```

Comment:
```suggestion
final class GlobalGroupsViewCell: UITableViewCell {
```
и название файла тоже, лучше чтобы все однинаковыми словами называлось, иначе кажется что разные сущности